src/files/FooterData.js

```diff
@@ -0,0 +1,69 @@
+const FindInformationItems = [
```

Comment:
Footer is not needed in this app. I think the only pieces of the site that are needed are.

- Breadcrumbs
- Sidebar Contact
- Sidebar Social

Reply:
done